src/Makefile.OCaml

```diff
@@ -129,6 +129,8 @@ ifneq ($(strip $(LDLIBS)),)
   CLIBS+=-cclib '$(LDLIBS)'
 endif
 
+CAMLCFLAGS+=-ccopt -D_FILE_OFFSET_BITS=64
```

Comment:
Why is that define ok in general, vs on some subset of systems?   That feels like a non-POSIX thing that has to be guarded, but I'm unsure.

Reply:
It will do no harm on systems that don't support this. This is required to request `off_t` being a 64-bit type. The new function calls added with this PR use `off_t` only on platforms which have this as a 64-bit type. It will do nothing bad on 32-platforms that don't support it (but you won't get large file support either).

The OCaml runtime itself (and by proxy unison) has relied on this since 2002, on all platforms, unconditionally.